src/std/build.ss

```diff
@@ -1,19 +1,11 @@
-#!/usr/bin/env gxi
+#!/usr/bin/env gxi-build-script
 ;; -*- Gerbil -*-
 
-(import "make" "build-config")
+(def srcdir (path-normalize (path-directory (this-source-file))))
+(def libdir (path-expand "lib" (getenv "GERBIL_HOME")))
+(add-load-path srcdir)
 
-(include "build-spec.ss")
-
-(let ((depgraph (call-with-input-file "build-deps" read))
-      (srcdir (path-normalize (path-directory (this-source-file))))
-      (libdir (path-expand "lib" (getenv "GERBIL_HOME"))))
-  (add-load-path (path-normalize (path-expand ".." srcdir)))
-  (make srcdir: srcdir
-        libdir: libdir
-        optimize: #t
-        static: #t
-        debug: 'src
-        depgraph: depgraph
-        prefix: "std"
-        build-spec))
+;; The indirection below allows the build-std to do its imports in a module context, transitively
+(import :gerbil/expander)
```

Comment:
is this import necessary? the expander is already loaded at this point.

Reply:
if you just want to call `import-module` you can do it with the canonical name without an import:
`gx#import-module`.